docs/index.md

```diff
@@ -86,6 +86,8 @@ Installing the latest prometheus plugin will create a config file (`~/.steampipe
 ```hcl
 connection "prometheus" {
   plugin = "prometheus"
+
+  # The address of your Prometheus (ignore if PROMETHEUS_URL env var is set).
```

Comment:
Is `PROMETHEUS_URL` a common env var used in any SDK, TF provider, etc.? The naming makes sense, but we like to stay consistent with what other CLI/tools/etc. use to make the plugin easier to use.

Reply:
So far, I haven't found any projects involving the use of environment variables for Prometheus.